examples/catalog/src/Reset.scss

```diff
@@ -0,0 +1,77 @@
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+  display: block;
+}
+body {
+  font-family: Roboto, sans-serif;
+}
+ol, ul {
+  list-style: none;
+}
+blockquote, q {
+  quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+  content: '';
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+a {
+  text-decoration: none;
+}
+select {
+  appearance: none;
```

Comment:
Do you actually need this? Shouldn't mdc-web's styles already have this.

Reply:
![image](https://user-images.githubusercontent.com/20078201/60847000-3bf1d800-a21c-11e9-9ef2-112cc01347c4.png)
After removing this line.
Native Select appears strange 😢

Reply:
hmm - seems like an mdc web bug. HOWEVER I think we're in the process of removing native select. We can ignore it for now.

Reply:
I agree @seriema...the user(developer using mdc) shouldn't have to do this to a component. But since we're deprecating the native select. I don't think we'll have time to get to it.

Reply:
Then I'll delete this line and remove Native Select.